src/components/views/audio_messages/PlaybackClock.tsx

```diff
@@ -76,7 +76,7 @@ export default class PlaybackClock extends React.PureComponent<IProps, IState> {
         }
         return <Clock
             seconds={seconds}
-            aria-live={this.state.playbackPhase === PlaybackState.Playing ? "off" : undefined}
+            role="timer"
```

Comment:
what is the rationale here? i'd expect us to still be suppressing this when playing state is off.

Reply:
a timer suppresses it magically for us